template/footer.tpl

```diff
@@ -17,7 +17,7 @@
 </div>
 {/if}
 {footer_script require='jquery'}
-document.cookie = 'screen_size='+jQuery(document).width()+'x'+jQuery(document).height()+';path={isset($COOKIE_PATH) ? $COOKIE_PATH : ''}';
+document.cookie = 'screen_size='+jQuery(document).width()+'x'+jQuery(document).height()+';path={$COOKIE_PATH|default:''}';

```

Comment:
This looks odd. Not sure if it does what the original code intended to do but failed. While it may not throw the error, this now consists of two Smarty "strings" `';path={$COOKIE_PATH|default:'` and `'}'` (where the `''` doubled single quotes do _not_ form an empty string to be used as default as it may seem) which _may_ get concatenated to one string `;path={$COOKIE_PATH|default:}` before evaluating and resulting in an empty default (not even empty string) by chance, but that's rather confusing. At worst Smarty silently errors at `;path={$COOKIE_PATH|default:` and appends a literal `}` but I have no clue of Smarty.

Reply:
I agree - with this commit pushed in the mean time (which I tested and works): https://github.com/Piwigo/piwigo-smartpocket/commit/738dcde354d73fad6b08442dcebdc9b5e8fbc002
I will cancel my PR